PickUpUI/src/App.js

```diff
@@ -3,6 +3,48 @@ import { RouterConfig } from './navigation/RouterConfig';
 import {Navigation} from "./components/Navigation"
 import Footer from './components/Footer';
 
+
+const restaurantsList = [
+  {
```

Comment:
Okay, so this will be removed after I finish implementing the endpoints for restaurant.